Tools/scripts/Utils/general_utils.py

```diff
@@ -1,4 +1,4 @@
-﻿"""Helper class for common operations."""
+"""Helper class for common operations."""
```

Comment:
This is an artifact from earlier changes... just reflecting what it is currently in the main v2 branch.